src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -137,177 +140,178 @@
      */
     public static final int CLASS_NAMEDCOLOR = 6;
 
+
     /**
      * ICC Profile Color Space Type Signature: 'XYZ '.
      */
-    public static final int icSigXYZData        = 0x58595A20;    /* 'XYZ ' */
+    public static final int icSigXYZData = 0x58595A20;
 
```

Comment:
Inline comment:
Looks like this type of documentation was used before we added javadoc for each constant.